File: 준영/프로그래머스/LV2/짝지어 제거하기.py
def solution(s):
    stack = []
    for i in range(len(s)):
        if len(stack) == 0:
            stack.append(s[i])
            continue

        if stack[-1] == s[i]:
            stack.pop()
        else:
            stack.append(s[i])

    if len(stack):
        answer = 0
    else:
        answer = 1

    return answer

File: 준영/프로그래머스/LV2/짝지어 제거하기.js
function solution(s) {
  var answer = 0
  var stack = []
  for (let i = 0; i < s.length; i++) {
    if (stack.length === 0) {
      stack.push(s[i])
      continue
    }

    if (stack[stack.length-1] === s[i]) {
      stack.pop()
    } else {
      stack.push(s[i])
    }
  }

  if (stack.length === 0) {
    answer = 1
  }

  return answer
}